### ISSUE TITLE
moment-timezone module not found

### ISSUE THREAD

**Issue by benvogan** [implydata/plyql#60]
I installed plyql via:

npm install -g plyql

But when I try to run it I get an error indicating that the moment-timezone module could not be found.  I had to install 0.8.15 to get around the problem.

**Comment by l15k4** [comment]
I got this one on 0.8.17 as well ... if I install `moment-timezone` then it wants `q`, and if I install `q` then : 
```
/usr/local/lib/node_modules/plyql/node_modules/plywood/build/plyqlParser.js:7640
    var originalColumns = Array.isArray(columns) ? columns.map((c) => c.name) 
                                                                    ^
SyntaxError: Unexpected token >
    at Module._compile (module.js:439:25)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/plyql/node_modules/plywood/build/plywood.js:12821:26)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
```

**Comment by vogievetsky** [comment]
should be fixed in the latest plyql and in the plyql released in Imply 2.0.0
